components/formats-gpl/src/loci/formats/in/LIFReader.java

```diff
@@ -1947,7 +2020,7 @@ else if (unit.equals("m")) {
             if (ms.sizeZ == 1) {
               ms.sizeZ = len;
               bytesPerAxis.put(nBytes, "Z");
-              physicalSizeZ = (physicalLen * len) / (len - 1);
+              physicalSizeZ = physicalLen;
```

Comment:
I think I fixed the issue with unreasonable acquisition dates, and also corrected a bug which overcompensated local time zones (compensation was done despite lif timestamps being in UTC to begin with).

As far as I understand, the timestamps we write in the OME-XML are still ambiguous though. loci.common.DateTools.convertDate calculates a UTC date, but I believe the returned string format "yyyy-MM-dd'T'HH:mm:ss" is typically interpreted as local time. But I am by no means an expert on this topic, and it is also probalby out of scope for this pull request.

Please let me know if I can do anything to help with the pixel size topic (e.g. provide example files) or anything else. Apologies for the long delay, I was ordered to finish some other assignment first.